src/partition/ReceivedPartition.cpp

```diff
@@ -135,10 +135,15 @@ void ReceivedPartition::compute()
                   _mesh->getName());
   }
 
+  // If any of the mappings is a scaled-scaled consistent mapping, extend filtering considering connectivity
+  _filterWithConnectivity = std::any_of(_fromMappings.begin(), _fromMappings.end(), [](const mapping::PtrMapping m) { return m->hasConstraint(mapping::Mapping::SCALEDCONSISTENT); }) or
+                            std::any_of(_toMappings.begin(), _toMappings.end(), [](const mapping::PtrMapping m) { return m->hasConstraint(mapping::Mapping::SCALEDCONSISTENT); });
```

Comment:
@uekerman Is it a good idea to always extend include connectivity in these cases?

Reply:
What do you mean by "always" here?

Reply:
When using scaled consistent mappings, the filter will always extend the selection using connectivity information.